package.json

```
@@ -177,6 +177,17 @@
 			{
 				"title": "NeuroPilot",
 				"properties": {
+					"neuropilot.access.inheritFromIgnoreFiles": {
+						"type": "boolean",
+						"default": true,
+						"description": "Allow NeuroPilot to read ignore patterns from an ignore file (e.g. .gitignore)."
+					},
+					"neuropilot.access.ignoreFilePath": {
+						"type": "array",
+						"default": [],
+						"description": "Optional path to a custom ignore file (relative or absolute). Takes priority over .gitignore if set."
```

```suggestion
						"description": "Optionally set paths to custom ignore files (relative or absolute)."
```

We shouldn't be prioritising .gitignore. Also, mention which files get priority (whether priority is last-first or top-first)